src/components/Card/Card.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from "react";
+
+const Card = () => {
+
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+
+    function login(e) {
```

Comment:
lembre de nomear suas funções com o máximo de legibilidade - clean code
```suggestion
    function handleFormOnSubmit(e) {
```